cluster-scope/base/core/namespaces/ds-black-flake/namespace.yaml

```diff
@@ -6,5 +6,8 @@ metadata:
     openshift.io/display-name: "Data Science: Black Flake"
     openshift.io/requester: data-science
     operate-first.cloud/issue: "https://github.com/operate-first/support/issues/76"
```

Comment:
Do we need both annotations here (lines 8 and 10)?
I noticed this in a couple other files so I''l just denote with: Duplicate field on line 10?